src/tests/ftest/pool/multi_server_create_delete_test.yaml

```diff
@@ -7,51 +7,30 @@ hosts:
 server_config:
   name: daos_server
 tests:
-  modes: !mux
-    modeall:
-      mode:
-        - 511
-        - PASS
-    readonly:
-      mode:
-        - 73
-        - FAIL
-    writeonly:
-      mode:
-        - 146
-        - PASS
-    execonly:
-      mode:
-        - 292
-        - PASS
-    modebogus:
-      mode:
-        - 111093499399299949949949949
-        - FAIL
-  uids: !mux
-    validuid:
-      uid:
+  users: !mux
+    validuser:
+      user:
         - valid
         - PASS
-    rootuid:
-      uid:
-        - 0
+    rootuser:
+      user:
+        - root
         - PASS
-    anotheruid:
+    anotheruser:
```

Comment:
Line 20 should use the same yaml path as the other entires in the mux: `uid:` -> `user:`